qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -2672,14 +2684,15 @@ def _copy_users(self, record_list: Dict):
     def check_lib_versions(self):
         """Check the stored versions of elemental and fractal"""
 
+        # check if versions table exist
+        if not self.engine.dialect.has_table(self.engine, "versions"):
+            return None
```

Comment:
Why doesn't this cause errors when creating a new database and connecting for the first time?

Reply:
This line checks if the table is created, and won't cause an error because there is no attempt to connect to any table or any kind of queries. This is used when you want to check the DB version, but you don't want to use ORM when tables are not created yet. So this line is important in the case of: 
`qcfractal-server init`, then `qcfractal-server start`